### ISSUE TITLE
Grafana dashboard

### ISSUE THREAD

**Issue by Merovius** [mdlayher/prombolt#2]
It would be cool if this would come with a json file for a Grafana dashboard :)

**Comment by mdlayher** [comment]
Good idea.  I don't actually have a dashboard prepared for this yet, but it's definitely something that should be done in the future.
